src/Shortener.hs

```diff
@@ -1,43 +1,51 @@
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes #-}
 
 module Shortener where
 
-import Control.Monad.IO.Class (MonadIO(liftIO))
+import Control.Monad.IO.Class (MonadIO (liftIO))
 import Data.Foldable (for_)
 import Data.IORef (modifyIORef, newIORef, readIORef)
 import Data.Map (Map)
 import qualified Data.Map as M
-import Data.Text (Text)
+import Data.Text (Text, pack, unpack)
 import qualified Data.Text.Lazy as LT
 import Network.HTTP.Types (status404)
 import Text.Blaze.Html.Renderer.Text (renderHtml)
 import qualified Text.Blaze.Html5 as H
 import qualified Text.Blaze.Html5.Attributes as A
+import Text.RE.Replace
+import Text.RE.TDFA.String
 import Web.Scotty
 
+onlyLetters :: String -> String
+onlyLetters s = replaceAll "" $ s *=~ [re|$([^a-zA-Z])|]
```

Comment:
Can you add an example of what the function does? What's the output for `https://github.com/`?

Reply:
Using this function, the output would be  `httpsgithubcom` but that's not what we intended to do. The goal was to validate the URL in order to make sure the user inputs a valid URL.